modules/longhorn/main.tf

```diff
@@ -0,0 +1,12 @@
+resource "helm_release" "longhorn" {
+  name             = "longhorn"
+  namespace        = "longhorn-system"
+  create_namespace = true
+  repository       = "https://charts.longhorn.io"
+  chart            = "longhorn"
+
+  /*  #Set overrides here  
+   set {
```

Comment:
We could consider allowing `helm`-based add-ons to take a `addon_config.chart_settings` (or something) that is a map of setting => value, and add dynamic `set` blocks based on that variable.